### PR TITLE
Desc/Asc last viewed date

### PR DESCRIPTION
**Pull request by yep-it-not-me**
Changes "Recently viewed" sorting option into two Desc/Asc "Last viewed date" options, consistent with other sorting options. Now folks can pick whatever they watched forever ago!

### CODE REVIEW

**Comment by yep-it-not-me** [comment]
Apologies for the extra commits, I'm working on windows and vscode, stuff I've never used for dev before. I'm also new to go.

File: ui/src/views/scenes/Filters.vue [hunk 1]
```diff
@@ -20,7 +20,8 @@
             <option value="rating_asc">↑ {{$t("Rating")}}</option>
             <option value="scene_added_desc">↓ {{$t("Scene added date")}}</option>
             <option value="scene_updated_desc">↓ {{$t("Scene updated date")}}</option>
-            <option value="last_opened">↻ {{$t("Recently viewed")}}</option>
+            <option value="last_opened">↓ {{$t("Last viewed date")}}</option>
```
Comment (SpedNSFW) [inline ui/src/views/scenes/Filters.vue:23]: Since you're adding a second sort order, you should keep with the pattern and add the `_desc` suffix here.

**Review by SpedNSFW — CHANGES_REQUESTED** [review]

**Comment by yep-it-not-me** [comment]
Closed because I couldn't figure out how to get github to update an existing PR ..

**Comment by SpedNSFW** [comment]
> Closed because I couldn't figure out how to get github to update an existing PR ..

Just push to the branch. Same way you got the `_desc` commit here.
[`07bea1c` (#392)](https://github.com/xbapps/xbvr/pull/392/commits/07bea1c21aa9938d0d5528c553e2c338bcb6a755)

**Pull request closed without merging**